src/pasttle/model.py

```diff
@@ -23,7 +23,7 @@ class Paste(Base):
 
     __tablename__ = 'paste'
 
-    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True, index=True, unique=True)
```

Comment:
I thought primary_key would set both of those other 2 flags automatically... I cannot find documentation supporting my assertion tho, I wonder if I dreamed it